server/test/app.test.js

```diff
@@ -0,0 +1,269 @@
+const { reset } = require("../db/reset");
+
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const app = require("../app.js");
+
+const { expect } = chai;
+
+chai.should();
+chai.use(chaiHttp);
+
+// Tests
+// 1) Create two users : main user and user's friend
+// 2) Send a follow request to user's friend using main user's token
+// 3) Accept the request using friends token
+// 4) Create new friendslist
+// 5) Create new poll
+// 6) Get poll with data
+// 7) Get poll using id
+// 8) Get invited polls (using friends token)
+// 9) Edit poll (update poll)
+// 10) Delete poll
+
+const expectToBe = (body, actual) => {
+  expect(body).to.satisfy((arr) =>
+    arr.some((poll) => expect(poll).to.deep.include(actual))
+  );
+};
+
+reset()
```

Comment:
Do we need to reset all DB every test run?

Reply:
User sign up can fail due to the possibility of user being already present from previous test runs or even manually. So I'm resetting all data.